src/page/legg-til-gjennomforing-tilgang/LeggTilGjennomforingTilgangPage.tsx

```diff
@@ -41,10 +49,11 @@ export const LeggTilGjennomforingTilgangPage = (): React.ReactElement => {
 				<TextField
 					label="Tiltaksnummer"
 					value={lopenrSokefelt}
+					onKeyPress={e => e.key === 'Enter' && handleOnSokClicked()}
 					onChange={e => {
 						const value = e.target.value
 
-						if (kunSiffer(value) || value === '')
+						if ((kunSiffer(value) && value.length <= 7) || value === '')
```

Comment:
Siden det begynner å bli et par sjekker så hadde det kanskje vært greit med en egen funksjon for `(kunSiffer(value) && value.length <= 7) || value === ''`, f.eks `erGyldigLopenr`